conflicting values for variable 'lat_bnds'

### What versions of software are you using?

-   Package Version: Main branch / Release 0.1.0

### What are the steps to reproduce this issue?
```
import xcdat

p = '/p/css03/scratch/cmip6/CMIP/NCC/NorCPM1/historical/r10i1p1f1/Amon/tas/gn/v20190914/*nc'
ds = xcdat.open_mfdataset(p)
```

> MergeError: conflicting values for variable 'lat_bnds' on objects to be combined. You can skip this check by specifying compat='override'.

### What happens? Any logs, error output, etc?

I think this is happening because the `lat_bnd` values have slight (e.g., 10<sup>-13</sup>) differences in the different netCDF files.

### Any other comments?
Opening with xarray works fine (as long as you do not set `data_vars="minimal"`), though the `lat_bnd` is larger than expected:

```
import xarray as xr
ds = xr.open_mfdataset(p)
print(ds.lat_bnds.shape)
```
> (2160, 142, 2)

It appears that the bounds differ depending on timestep:
```
np.array(ds.lat_bnds[0] - ds.lat_bnds[-1])
```

> array([[-9.00000000e+01, -8.90526316e+01],
>           [-8.90526316e+01, -8.71578947e+01],
>        [-8.71578947e+01, -8.52631579e+01],
> ...
>        [            nan,             nan],
>        [ 8.90526316e+01,  9.00000000e+01],
>        [            nan,             nan]])

I'm not totally sure why there are `NaN` values. They don't appear when I use `ncdump -v lat_bnds /p/css03/scratch/cmip6/CMIP/NCC/NorCPM1/historical/r10i1p1f1/Amon/tas/gn/v20190914/tas_Amon_NorCPM1_historical_r10i1p1f1_gn_185001-201412.nc`, but they do appear after loading the dataset with xarray. 

I do not know what to do with this problem, though CDAT seems to be able to load the bounds with no problem.

I think nan values are being produced because there are very small differences in lat_bnds values as you mentioned (also shown in code example below) and an **outer join (union of object indexes)** is performed by default. These nan values subsequently increase the size of lat_bnds from (96, 2) to (142, 2) after calling `open_mfdataset()`.

Comparing shapes of lat_bnds between datasets and performing floating point comparison:
```
p1 = f"{path}tas_Amon_NorCPM1_historical_r10i1p1f1_gn_185001-201412.nc"
p2 = f"{path}tas_Amon_NorCPM1_historical_r10i1p1f1_gn_201501-201812.nc"
p3 = f"{path}tas_Amon_NorCPM1_historical_r10i1p1f1_gn_201901-202912.nc"

>>> ds1 = xr.open_dataset(p1)
>>> ds2 = xr.open_dataset(p2)
>>> ds3 = xr.open_dataset(p3)
>>> ds1.lat_bnds.shape
(96, 2)
>>> ds2.lat_bnds.shape
(96, 2)
>>> ds3.lat_bnds.shape
(96, 2)

>>> np.testing.assert_allclose(ds1.lat_bnds, ds2.lat_bnds)
>>> np.testing.assert_allclose(ds1.lat_bnds, ds3.lat_bnds)
AssertionError: 
Not equal to tolerance rtol=1e-07, atol=0

Mismatched elements: 2 / 192 (1.04%)
Max absolute difference: 0.
Max relative difference: 0.
 x: array([[-90.      , -89.052632],
       [-89.052632, -87.157895],
       [-87.157895, -85.263158],...
 y: array([[-90.      , -89.052632],
       [-89.052632, -87.157895],
       [-87.157895, -85.263158],...


>>>np.testing.assert_allclose(ds2.lat_bnds, ds3.lat_bnds)
True
```

Comparing difference joins since I noticed that the lat_bnds of the individual datasets had a shape of (96, 2), not (142, 2):
```
# “outer”: use the union of object indexes (default xarray value)
# ==========================================================
>>> ds = xcdat.open_mfdataset(
    p, data_vars="minimal", coords="minimal", compat="override", join="outer"
)
>>> ds.lat_bnds.shape
(142, 2)
>>> ds_left.lat_bnds.isnull().values
Has null values

# "left": use indexes from the first object with each dimension
# ==========================================================
>>> ds_left = xcdat.open_mfdataset(
    p, data_vars="minimal", coords="minimal", compat="override", join="left"
)
>>> ds_left.lat_bnds.shape
(96, 2)
>>> ds_left.lat_bnds.isnull().values
No null values

# “override”: if indexes are of same size, rewrite indexes to be those of the first object with that dimension. Indexes for the same dimension must have the same size in all objects.
# ==========================================================
>>> ds_override = xcdat.open_mfdataset(
    p, data_vars="minimal", coords="minimal", compat="override", join="override"
)
>>> ds_left.lat_bnds.shape
(96, 2)
>>> ds_override.lat_bnds.isnull().values
No null values

# %%
>>> ds_left.lat_bnds.identical(ds_override.lat_bnds)
True
```

The possible solutions I found so far are to use `join="left"` or `join="override"` in `open_mfdataset()`, but there may be implications with this option (e.g., if the first dataset has missing values and you use them for the single joined dataset).

--- a/docs/faqs.rst
+++ b/docs/faqs.rst
@@ -5,7 +5,7 @@ Frequently Asked Questions
 How do I open files that have conflicting values for a data variable(s)?
 ------------------------------------------------------------------------
 In xarray, the default setting for checking compatibility across files is ``compat='no_conflicts'``.
-If conflicting values exist for a data variable between the files, xarray raises ``MergeError: conflicting values for variable <DATA VAR NAME> on objects to be combined. You can skip this check by specifying compat="override".``
+If conflicting values for a data variable between the files exists, xarray raises ``MergeError: conflicting values for variable <DATA VAR NAME> on objects to be combined. You can skip this check by specifying compat="override".``
 
 Let's say you try opening two files using ``xcdat.open_mfdataset()`` and the aforementioned ``MergeError`` appears for the ``lat_bnds`` data var.
 You perform a floating point comparison between both files for ``lat_bnds`` and find a very small floating point difference at specific coordinates.

--- a/docs/faqs.rst
+++ b/docs/faqs.rst
@@ -1,0 +1,22 @@
+==========================
+Frequently Asked Questions
+==========================
+
+How do I open files that have conflicting values for a data variable(s)?
+------------------------------------------------------------------------
+In xarray, the default setting for checking compatibility across files is ``compat='no_conflicts'``.
+If conflicting values exist for a data variable between the files, xarray raises ``MergeError: conflicting values for variable <DATA VAR NAME> on objects to be combined. You can skip this check by specifying compat="override".``
+
+Let's say you try opening two files using ``xcdat.open_mfdataset()`` and the aforementioned ``MergeError`` appears for the ``lat_bnds`` data var.
+You perform a floating point comparison between both files for ``lat_bnds`` and find a very small floating point difference at specific coordinates.
+
+To workaround this data quality issue and proceed with opening the files, pass these keyword arguments:
+
+1. ``compat="override"``: skip comparing and pick variable from first dataset
+2. ``join="override"``:  if indexes are of same size, rewrite indexes to be those of the first object with that dimension. Indexes for the same dimension must have the same size in all objects.
+
+   - ``join`` is set to `"outer_join"`, which might not be desired.
+
+``xcdat.open_mfdataset('path/to/files/*.nc', compat="override", join="override")``
+
+More information here: https://xarray.pydata.org/en/stable/generated/xarray.open_mfdataset.html#xarray-open-mfdataset

--- a/docs/faqs.rst
+++ b/docs/faqs.rst
@@ -15,7 +15,7 @@ To workaround this data quality issue and proceed with opening the files, pass t
 1. ``compat="override"``: skip comparing and pick variable from first dataset
 2. ``join="override"``:  if indexes are of same size, rewrite indexes to be those of the first object with that dimension. Indexes for the same dimension must have the same size in all objects.
 
-   - ``join`` is set to `"outer_join"`, which might not be desired.
+   - ``join`` is set to `"outer_join"` by default, which might not be desired.
 
 ``xcdat.open_mfdataset('path/to/files/*.nc', compat="override", join="override")``
 

--- a/docs/faqs.rst
+++ b/docs/faqs.rst
@@ -5,10 +5,10 @@ Frequently Asked Questions
 How do I open files that have conflicting values for a data variable(s)?
 ------------------------------------------------------------------------
 In xarray, the default setting for checking compatibility across files is ``compat='no_conflicts'``.
-If conflicting values for a data variable between the files exists, xarray raises ``MergeError: conflicting values for variable <DATA VAR NAME> on objects to be combined. You can skip this check by specifying compat="override".``
+If conflicting values for a data variable exists between the files, xarray raises ``MergeError: conflicting values for variable <DATA VAR NAME> on objects to be combined. You can skip this check by specifying compat="override".``
 
 Let's say you try opening two files using ``xcdat.open_mfdataset()`` and the aforementioned ``MergeError`` appears for the ``lat_bnds`` data var.
-You perform a floating point comparison between both files for ``lat_bnds`` and find a very small floating point difference at specific coordinates.
+You perform floating point comparison for ``lat_bnds`` and find a very small difference at specific coordinates.
 
 To workaround this data quality issue and proceed with opening the files, pass these keyword arguments:
 

--- a/docs/index.rst
+++ b/docs/index.rst
@@ -85,6 +85,7 @@ See `LICENSE <LICENSE>`_ for details
    Getting Started <getting_started>
    API Reference <api>
    Xarray API Reference <https://xarray.pydata.org/en/stable/api.html>
+   Frequently Asked Questions <faqs>
 
 .. toctree::
    :maxdepth: 2

